src/poetry/installation/executor.py

```diff
@@ -588,7 +588,7 @@ def _install_git(self, operation: Install | Update) -> int:
 
         src_dir = self._env.path / "src" / package.name
         if src_dir.exists():
-            safe_rmtree(str(src_dir))
+            remove_directory(str(src_dir))
```

Comment:
Any reason to cast to string? And shouldn't there be a `force=True` just like in the other cases?